pkg/reconciler/kubernetes/tektoninstallerset/tektoninstallerset.go

```diff
@@ -136,6 +136,16 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, installerSet *v1alpha1.T
 	// Update Status for Deployment Resources
 	installerSet.Status.MarkDeploymentsAvailable()
 
+	// Install Job Resources
```

Comment:
can we keep the order same, create jobs before deployment so that it will be same order like before and we dont end up in any regression situation

Reply:
thanks